scripts/get_gnomad_lof_variants.py

```diff
@@ -181,6 +176,17 @@ def variant_id(locus, alleles):
     )
 
 
+def liftover_variant_id(exome_liftover_id, genome_liftover_id, alleles):
+    return variant_id(
+        hl.if_else(
+            hl.is_missing(exome_liftover_id),
+            genome_liftover_id,
```

Comment:
Are both of these meant to be `genome_liftover_id` or is line 184 perhaps supposed to be `exome_liftover_id`?

Reply:
Ohp yes great catch, thank you! My mistake.